src/app.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+var todoItems = [];
+
+class TodoList extends React.Component {
+  render () {
+    var items = this.props.items.map((item, index) => {
```

Comment:
Prefer `let` and `const` to `var`. They were added to the language because `var` has some subtle scoping issues and it can catch you by surprise.